dashboards-observability/public/components/event_analytics/explorer/log_patterns/patterns_table.tsx

```diff
@@ -25,13 +25,11 @@ interface PatternsTableProps {
   tabId: string;
   query: any;
   isPatternLoading: boolean;
-  totalHits?: number;
 }
 
 export function PatternsTable(props: PatternsTableProps) {
-  const { tableData, tabId, onPatternSelection, query } = props;
-  const patternsData = useSelector(selectPatterns)[tabId];
```

Comment:
was patternsData just never used?

Reply:
patternsData used to contain the results of `stats count()` as total hit, forgot to remove it since i removed `stats count()` call